docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.5'
+
+services:
+
+  qmk_api:
+    build: .
+    ports:
+     - "5000:5000"
+    environment:
+      S3_ACCESS_KEY: minio_dev
+      S3_SECRET_KEY: minio_dev_secret
+      REDIS_HOST: redis
+      S3_HOST: http://minio:9000
+      UPDATE_API: 'true'
+
```

Comment:
Add
```yaml
    volumes:
      - .:/qmk_api
```
to enable live code updating when developing locally.

Reply:
Ooohhh, awesome, I'll play with this later today.

Reply:
haven't gotten to play with this yet, but with the new structure I think it should be
```
        volumes:
          - ./qmk_api:/qmk_api
```
or something